src/components/application_manager/rpc_plugins/sdl_rpc_plugin/include/sdl_rpc_plugin/commands/mobile/perform_interaction_request.h

```diff
@@ -232,7 +259,25 @@ class PerformInteractionRequest
    */
   void SendBothModeResponse(const smart_objects::SmartObject& msg_param);
 
+  /**
+   * @brief Sends UiClosePopUp request to HMI
+   */
+  void SendClosePopupRequestToHMI();
+
+  /**
+   * @brief Sets the choice according to the current interaction mode and first
+   * received choice id (UI or VR).
+   *
+   * @param msg_param Message parameters which will be included in the response
+   * to mobile device.
+   */
+  bool SetChoiceIdToResponseMsgParams(
```

Comment:
We should add @return documentation here

Reply:
@iCollin added in https://github.com/smartdevicelink/sdl_core/pull/3122/commits/5e77a65c1a12eff7d2aef300861d5936fba60fc9